### PR TITLE
code UI detail Screen

### PR DESCRIPTION
**Pull request by vitconpc**
Code UI detail Screen
https://edu-redmine.framgia.vn/issues/1519

### CODE REVIEW

**Comment by vitconpc** [comment]
ready

File: app/src/main/res/layout/activity_detail.xml [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                    app:layout_constraintStart_toEndOf="@+id/image_ic_detail_rating"
+                    app:layout_constraintTop_toBottomOf="@+id/text_detail_title" />
+
+                <TextView
+                    android:id="@+id/text_detail_rating_value"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginLeft="@dimen/dp_8"
+                    android:layout_marginRight="8dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_detail.xml]: 95 97 hardcode

File: app/src/main/res/layout/item_cast.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                android:paddingStart="@dimen/dp_15"
+                android:text="@{cast.character}"
+                android:textAppearance="@style/TextAppearance.AppCompat.Small"
+                android:textColor="@android:color/white"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent" />
+        </android.support.constraint.ConstraintLayout>
+    </android.support.v7.widget.CardView>
+</layout>
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_cast.xml]: need an endline

File: app/src/main/res/layout/item_film.xml [hunk 3]
```diff
@@ -14,14 +14,14 @@
...
             android:layout_width="@dimen/dp_172"
             android:layout_height="@dimen/dp_193"
-            android:layout_marginEnd="@dimen/dp_8"
-            android:layout_marginLeft="@dimen/dp_8"
-            android:layout_marginRight="@dimen/dp_8"
-            android:layout_marginStart="@dimen/dp_8"
+            android:layout_marginEnd="8dp"
+            android:layout_marginLeft="8dp"
+            android:layout_marginRight="8dp"
+            android:layout_marginStart="8dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_film.xml]: 21 -> 24 hardcode

**Comment by tuannguyen38** [comment]
commented

**Pull request merged**